### PR TITLE
User can choose starting point by input

### PR DESCRIPTION
**Pull request by EevanR**
[PT LINK](https://www.pivotaltracker.com/story/show/171051267)

USER STORY

As a User
In order to receive my trip package
I need to be able to choose my starting point

- add create trip component
- add input field for city
- send input value to backend

### CODE REVIEW

File: src/components/Destination.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+import { sendDestination } from "../modules/destination";
+
+const SendDestination = () => {
+  let [message, setMessage] = useState("")
+
+  const submitPlace = async (e) => {
+    e.preventDefault();
+    const response = await sendDestination(
```
Comment (oliverochman) [inline src/components/Destination.jsx]: why are we sending off the location here but not when the user picks a location on the map?

File: cypress/integration/userCanSetDestination.feature.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  });
+
+  it("can submit destination successfully", () => {
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/**",
+      response: 200
+    });
+
+    cy.route({
```
Comment (oliverochman) [inline cypress/integration/userCanSetDestination.feature.js:9]: why are you stubbing out a "GET" request?

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: src/components/Destination.jsx (hunk 1 above)
Comment (EevanR) [reply src/components/Destination.jsx]: We've had a bit of a discussion, and this is going to be re worked a bit

File: cypress/integration/userCanSetDestination.feature.js (hunk 2 above)
Comment (EevanR) [reply cypress/integration/userCanSetDestination.feature.js:9]: things have changed now !!!! this doesnt exist

**Review by mantanquzzmo — APPROVED** [review]
lgtm

**Pull request merged**